src/atoms/footer/__snapshots__/footer.test.tsx.snap

```diff
@@ -1,13 +1,3 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Footer should render 1`] = `
```

Comment:
Add tests for:
- when nothing is passed (default)
- when footer is disabled and `showPoweredByQt` is a boolean
- when footer is disabled and `showPoweredByQt` is a function
- negative test cases